stable/artifactory-ha/CHANGELOG.md

```diff
@@ -1,6 +1,16 @@
 # JFrog Artifactory-ha Chart Changelog
 All changes to this chart will be documented in this file.
 
+<<<<<<< HEAD
```

Comment:
@amithins you have a conflict here. Please fix it

Reply:
Fixed